testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -217,13 +222,17 @@ impl RunLoop {
         .unwrap();
         coordinator_dispatcher.dispatch_boot_receipts(receipts);
 
+        let atlas_config = AtlasConfig::default();
+        let moved_atlas_config = atlas_config.clone();
+
         thread::spawn(move || {
             ChainsCoordinator::run(
                 chain_state_db,
                 coordinator_burnchain_config,
                 attachments_tx,
                 &mut coordinator_dispatcher,
                 coordinator_receivers,
+                moved_atlas_config,
```

Comment:
why not just have `atlas_config.clone()` here?

Reply:
The `atlas_config` would be moved to the another thread.